man/systemd-firstboot.xml

```diff
@@ -34,12 +34,22 @@
   <refsect1>
     <title>Description</title>
 
-    <para><command>systemd-firstboot</command> initializes basic system settings interactively during the
-    first boot, or non-interactively on an offline system image. The service is started during boot if
-    <varname>ConditionFirstBoot=yes</varname> is met, which essentially means that <filename>/etc/</filename>
-    is unpopulated, see
+    <para>The <command>systemd-firstboot.service</command> is one of the units which are used to initialize
```

Comment:
This is weird, I think this should say "The systemd-firstboot service" instead of "The systemd-firstboot.service"

Reply:
Unfortunately there is no "system-firstboot service". The actual name is `systemd-firstboot.service`.

Reply:
With more details: it's important to include the actual name of the unit. It's important for grepping and being precise. In this particular case, the binary is only used in one unit, but there are other cases where the same binary is used in multiple units, and we shouldn't use the name of the binary as the shorthand for the unit name.